scripts/catalog-tests/manage-test-workspaces.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+usage() {
+  echo "usage: $0 <env> <create/delete>"
+  exit 1
+}
+
+create_billing_project() {
+  curl -X 'POST' \
+    -H 'Content-Type: application/json' \
+    -H "${AUTH_TOKEN}" \
+    -d "{ \"projectName\": \"${PROJECT_NAME}\", \"billingAccount\": \"${BILLING_ACCT}\" }" \
+    "${RAWLS_ENV}/billing/v2"
+}
+
+create_workspace() {
+  curl -s -X 'POST' \
+    -H 'Content-Type: application/json' \
+    -H "${AUTH_TOKEN}" \
+    -d "{ \"namespace\": \"${PROJECT_NAME}\", \"name\": \"${WORKSPACE_NAME}\", \"attributes\": {} }" \
+    "${RAWLS_ENV}/workspaces" | jq -r .workspaceId
+}
+
+create_action() {
+  create_billing_project
+  create_workspace
+}
+
+delete_billing_project() {
+  curl -X 'DELETE' -H "${AUTH_TOKEN}" \
+    "${RAWLS_ENV}/billing/v2/${PROJECT_NAME}"
+}
+
+delete_workspace() {
+  curl -X 'DELETE' -H "${AUTH_TOKEN}" \
+    "${RAWLS_ENV}/api/workspaces/${PROJECT_NAME}/${WORKSPACE_NAME}"
+}
+
+delete_action() {
+  delete_workspace
+  delete_billing_project
+}
+
+AUTH_TOKEN=$(printf "Authorization: Bearer %s" "$(gcloud auth print-access-token)")
```

Comment:
Is there a reason to use `printf` here? You can expand the subshell output directly into the string:
```suggestion
AUTH_TOKEN="Authorization: Bearer $(gcloud auth print-access-token)"
```